.github/workflows/test.yml

```diff
@@ -18,16 +17,14 @@ jobs:
       # keep running so we can see if tests with other k3s/k8s/helm versions pass
       fail-fast: false
       matrix:
-        include:
-            - k3s-channel: v1.19
-              helm-version: v3.5.0
-              test: main
-            - k3s-channel: v1.19
-              helm-version: v3.5.0
-              test: auth
-            - k3s-channel: v1.19
-              helm-version: v3.5.0
-              test: helm
+        k3s-channel:
```

Comment:
Instead of describing each case separately it's possible to just list all the options and then Github will run a build for every possible combination.